### DISCUSSION TITLE
参与开发与沟通

### DISCUSSION THREAD

**Discussion by coljiang** [swuecho/chat#352] [General]
您好,
      项目很棒!感谢付出.
      请问有qq或微信群么?我也是一名go开发想参与开发,完成TODO.并在群里和你沟通.

**Comment by swuecho** [comment]
感谢你的认可， 目前没有群， 我建一个。

**Reply by swuecho** [reply]
查看一下你gmail 邮箱或者发我你wechat id

**Reply by renwofei423** [reply]
群号多少？

**Reply by swuecho** [reply]
QQ: 603914071

**Comment by swuecho** [comment]
QQ: 603914071 

有问题群里问这里的论坛都可以.

**Comment by swuecho** [comment]
对于参与, 提出你的需求, 或者直接弄pr 都可以. 

建议先弄提个需求, 大家先讨论一下. 

目前我这边时间有限,  希望更多人参与, 如果有前端或者UI 参与进来更好.

对于参与的人, 如果你需要用chatgpt 3.5, 我这边可以个送我自建的网站账号.

**Comment by swuecho** [comment]
我最近考虑过做embedding 以及整合langchain 相关的功能, 但是时间有限,这都是需要比较多时间的.
